python/paddle/fluid/dygraph/jit.py

```diff
@@ -54,70 +53,122 @@ def extract_vars(inputs):
     return result_list
 
 
-def _dygraph_to_static_code_(dygraph_func):
+def _dygraph_to_static_func_(dygraph_func):
+    """
+    Converts imperative dygraph APIs into declarative function APIs. Decorator
+    @dygraph_to_static_func only converts imperative dygraph APIs into
+    declarative net-building APIs, which means it doesn't return immediate
+    digital result as imperative mode. Users should handle Program and Executor
+    by themselves.
+
+    Note:
+    This decorator is NOT our recommended way to transform imperative function
+    to declarative function. We will remove this decorator after we finalize
+    cleaning up code.
+
+    Args:
+        dygraph_func (callable): callable imperative function.
+
+    Returns:
+        callable: converting imperative dygraph APIs into declarative
+        net-building APIs.
```

Comment:
Better to capitalize the  first letter of sentence in `Args` and `Returns`.

Reply:
Done